src/bashly.yml

```diff
@@ -58,7 +77,16 @@ dependencies:
   convert:
     command: [convert]
     help: "Please install imagemagick package"
+  smmstoretool:
+    command: [cbfstool]
+    help: >
+      You can build one from source code:
+        https://github.com/Dasharo/dcu#compiling-smmstoretool
```

Comment:
Links do not work

Reply:
Please recheck.

Reply:
Fixed in https://github.com/Dasharo/dcu/pull/5/commits/6303349354ed85feb7bd5c78462af992d32fdad3